pegasus/helpers/hoc_helpers.rb

```diff
@@ -5,6 +5,12 @@
 UNSAMPLED_SESSION_ID = 'HOC_UNSAMPLED'
 CARTOON_NETWORK = 'CN'
 
+ACTIVITY_TRACKING_SOURCES = {
+  HOC: 'hoc',
+  LEARN: 'learn',
+  PIXEL: 'pixel'
+}.freeze
```

Comment:
I think the hash is preferable just because the constant `HOC` doesn't have nearly as much context as `ACTIVITY_TRACKING_SOURCES[:HOC]`